index.js

```diff
@@ -8,6 +8,7 @@ const config = require('./config');
 
 const m = options => {
 	options = Object.assign({
+		config,
```

Comment:
This is really useful; it allows us to override `config` for tests so the results only include the specific plugins we want to test.

Without this, it's difficult to assume where in the result message set a potential error will be as we add and update other rules.